src/app/getting-started/getting-started.component.html

```diff
@@ -120,9 +120,8 @@ <h2 class="circle">2</h2>
           <div class="success-border">
             <h2 class="success-title">Success! You're ready to start in OpenShift.io.</h2>
             <p>
-              Go to your <a [routerLink]="['/', '_home']">home</a> page to start coding, plan work, or do other things.
-              You can also fill out more information about you in your
-              <a [routerLink]="['/', '_home']">profile</a> so that teammates can identify you.
+              Go to your home page to start coding, plan work, or do other things.
+              You can also fill out more information about you in your profile so that teammates can identify you.
```

Comment:
New text now that we had to remove the links should be:
"You can now proceed to your home page to plan work, code, and deploy your applications."